ibis/backends/clickhouse/__init__.py

```diff
@@ -585,14 +583,17 @@ def create_table(
         Table
             The new table
         """
-        if temp:
-            raise com.IbisError(
-                "ClickHouse temporary tables are not yet supported due to a bug in `clickhouse_driver`"
-            )
-
         tmp = "TEMPORARY " * temp
         replace = "OR REPLACE " * overwrite
-        table = self._fully_qualified_name(name, database)
+
+        if temp and overwrite:
+            raise com.IbisInputError("Cannot specify both temp and overwrite")
```

Comment:
ClickHouse currently does not allow `CREATE OR REPLACE TEMPORARY`.

We could probably implement this ourselves using `ALTER`, but that can be done in a follow-up.